roles/nfd_deploy/tasks/main.yml

```diff
@@ -12,10 +28,36 @@
   args:
     warn: false # don't warn about using sed here
 
+- name: "Wait for the NFD Operator OperatorHub ClusterServiceVersion"
+  command:
+    oc get ClusterServiceVersion
+       -oname
+       -n openshift-nfd
+  register: nfd_operator_csv_name
```

Comment:
@kpouget I hit another issue here when I have multiple operators installed in namespace openshift-nfd, `nfd_operator_csv_name` will contain more than one line.

Reply:
hum, I see, you see all the operators install in "All the namespaces"
```
oc get ClusterServiceVersion -n openshift-nfd -oname
clusterserviceversion.operators.coreos.com/gpu-operator-certified.v1.6.0
clusterserviceversion.operators.coreos.com/nfd.4.6.0-202103010126.p0
```
I'll fix it tomorrow, thanks for reporting it!

Reply:
@BumbleFeng 
it will be fixed soon (CI testing on-going), see this PR and comment:
https://github.com/openshift-psap/ci-artifacts/pull/126#issuecomment-829268571